plugins/star-rating/api/api.js

```diff
@@ -119,6 +437,7 @@ var plugin = {},
         return false;
     });
 
```

Comment:
Since there is data in separate collection with uid, per GDPR you need to handle 3 more events:

Providing examples here:
1. device id change: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L19-L27
2. app user delete: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L28-L35
3. app user data export: https://github.com/Countly/countly-server/blob/master/plugins/crashes/api/api.js#L37-L48